pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -201,28 +201,7 @@ func Run() {
 	pubBaseOsStatus.SignalRestarted()
 	log.Infof("SignalRestarted done")
 
-	// Need stillRunning across handleInitUpdateVerifiedObjects since
-	// it reverifies potentially huge images
-	keepRunning := func(doneChan chan struct{}) {
-		log.Infof("keepRunning starting")
-		for {
-			select {
-			case <-doneChan:
-				log.Infof("keepRunning done")
-				close(doneChan)
-				return
-
-			case <-stillRunning.C:
-			}
-			agentlog.StillRunning(agentName, warningTime, errorTime)
-		}
-	}
-	doneChan := make(chan struct{})
-	go keepRunning(doneChan)
-
-	handleInitUpdateVerifiedObjects(&ctx)
-
-	doneChan <- struct{}{}
+	// handleInitUpdateVerifiedObjects(&ctx)
```

Comment:
That doesn't look like a comment, nor like code ;-)

Reply:
Will remove these functions..

Reply:
Please remove the functions which you commented as dead as well.